src/main/groovy/jenkins/automation/builders/CheckmarxSecurityJobBuilder.groovy

```diff
@@ -35,75 +25,22 @@ class CheckmarxSecurityJobBuilder {
     String name
     String description
     List<String> emails
-    def scanRepo = []
-    String checkmarxComment
-    Boolean useOwnServerCredentials = false
-    String serverUrl
-    String username
-    String password
-    String groupId
-    Boolean incremental = true
-    String filterPattern = "!**/_cvs/**/*, !**/.svn/**/*"
-    String excludeFolders = "resources, .git"
-    String preset = "17"// Default 2014
-    Boolean presetSpecified = false
-    String fullScanCycle = "10"
-    Boolean vulnerabilityThresholdEnabled = true
-    String highThreshold = 1
-    String mediumThreshold = 2
-    String lowThreshold = 3
+    List scanRepo = []
```

Comment:
Does this parameter actually take multiple repos? What's the behavior here? All of our examples have just one repo.

Reply:
It does -- and this is not a change from before. See my PR in the operations-jenkins-automation repo for an example with multiple repos.